src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/object_array_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Symfony\Component\DependencyInjection\Tests\Fixtures\Bar;
+use Symfony\Config\ServicesConfig;
```

Comment:
It would be the same, but should it use the `Symfony\Config` namespace?
```suggestion
namespace Symfony\Config;

use Symfony\Component\DependencyInjection\Tests\Fixtures\Bar;
```